lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1154,6 +1154,9 @@ def _create_node_runbook(
             azure_node_runbook.shared_gallery = self._parse_shared_gallery_image(
                 azure_node_runbook.location, azure_node_runbook.shared_gallery
             )
+            azure_node_runbook.hyperv_generation = int(
```

Comment:
Not add it here, refer to the marketplace image, the logic should be after L1183.

Reply:
updated